src/lib/states/__tests__/task.test.js

```diff
@@ -134,7 +134,7 @@ describe.skip('Test lambda task', () => {
   });
 });
 
-describe('Test activity task', () => {
+describe.skip('Test activity task', () => {
```

Comment:
reason?

Reply:
Will try to use with https://www.npmjs.com/package/aws-step-father to be able to send task heartbeats/failure/success messages.